CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 cmake_minimum_required(VERSION 3.13...3.18)
 
+if(NOT CMAKE_Fortran_COMPILER)
+    set(CMAKE_Fortran_COMPILER "/usr/bin/gfortran")
+endif()
+
```

Comment:
```suggestion
```

This should not be hardcoded. CMAKE_Fortran_COMPILER is set automatically by the `project(` cmake command. If cmake needs help finding fortran, this should be done outside of the cmake files, e.g. by using the FC environment variable. My gfortran doesn't live in /usr/bin so this will break my workflow.